ee/candi/cloud-providers/dynamix/layouts/standard/base-infrastructure/main.tf

```diff
@@ -1,2 +1,23 @@
 # Copyright 2024 Flant JSC
 # Licensed under the Deckhouse Platform Enterprise Edition (EE) license. See https://github.com/deckhouse/deckhouse/blob/main/ee/LICENSE
+
+data "decort_account" "account" {
+   account_id = local.account_id
+}
+
+resource "decort_resgroup" "resource_group" {
```

Comment:
Rename to `decort_resourse_group`

Reply:
It's a redundant name. We already know it's decort api:
```terraform
  rg_id = decort_resgroup.resource_group.rg_id
```

Reply:
The name `decort_resourse_group` is more readable than the name `decort_resgroup`

Reply:
ok